### PR TITLE
Prettify code after generate

### PR DESCRIPTION
**Pull request by zulucoda**
This feature fixes this issue: #9 

**Results:**
screenshot from 2018-07-28 18-55-08_formatted

**Test results with formatted code:**
screenshot from 2018-07-28 19-09-46

### CODE REVIEW

File: src/test/generator-utilities.spec.js [hunk 1]
```diff
@@ -72,6 +73,60 @@ describe('Generator Utilities - Unit Test', () => {
...
+            };
+
+            const testPlaceHolder = getAllPlaceholderNames(name);
+
+            const prettierSpy = jest.spyOn(prettier, 'format');
+
+            function assert() {
+                expect(prettierSpy).toHaveBeenCalled();
+                prettierSpy.mockRestore();
+                done();
```
Comment (JPStrydom) [inline src/test/generator-utilities.spec.js]: 🥇

File: src/generator-utilities.js [hunk 2]
```diff
@@ -152,7 +153,16 @@ function createTemplate(directory, placeholderNames, omitComments, callback) {
...
+        const prettierParser = directory.generated.indexOf('.scss') > -1 ? 'scss' : 'babylon';
+
+        const formattedCode = prettier.format(data, {
+            tabWidth: 4,
+            singleQuote: true,
+            printWidth: 100,
+            parser: prettierParser
+        });
+
+        fs.writeFile(directory.generated, formattedCode, err => {
```
Comment (JPStrydom) [inline src/generator-utilities.js]: Nice work dude!

Comment (zulucoda) [reply src/generator-utilities.js]: @JPStrydom thanks, dude. I like the config feature you added. I'm thinking of moving default settings to config and allow advanced users to specify their prettier settings. Do you think it gonna be overkill?

Comment (JPStrydom) [reply src/generator-utilities.js]: I think we can take it even one step further... We can allow users to specify their own templates as well. We already have the `grcc.json` file that we can use to specify all kinds of config. But I think it's a nice to have. Once users start requesting it we can add it. What do you think?

**Pull request merged**